libraries/crypto/src/ecdsa.rs

```diff
@@ -257,6 +261,10 @@ impl PubKey {
         self.p.gety().to_int().to_bin(y);
     }
 
+    /// Verifies if the data's hash matches its signature.
+    ///
+    /// This function is not a constant time implementation, and does not resist side channel
+    /// attacks. Only use if all data invovled is public knowledge.
```

Comment:
thanks for adding doc :)

typo: involved

Reply:
Thanks, fixed.